well-architected/devops/overview.md

```diff
@@ -18,7 +18,7 @@ categories:
 
 # Overview of the operational excellence pillar
 
-The operational excellence pillar covers the operations processes that keep an application running in production. Deployments must be reliable and predictable. Automated deployments reduce the chance of human error. Fast and routine deployment processes won't slow down the release of new features or bug fixes. Equally important, you must be able to quickly roll back or roll forward if an update has problems.
+The operational excellence pillar covers the operations processes that keep a workload running in production. Deployments must be reliable and predictable. Automated deployments reduce the chance of human error. Fast and routine deployment processes won't slow down the release of new features or bug fixes. Equally important, you must be able to quickly roll back or roll forward if an update has problems.
```

Comment:
Changing to "a workload" as based on customer feedback not every team using WAF OpEx also builds an application. I thought this term was more inclusive.